backend/app/services/adapters/task_kinds/helpers.py

```
@@ -539,6 +539,28 @@ def build_lite_task_list(
+            )
+            .filter(
+                ResourceMember.resource_type == ResourceType.TASK.value,
+                ResourceMember.resource_id.in_(task_ids_for_members),
+                ResourceMember.status == MemberStatus.APPROVED.value,
+                # Exclude share records (copied_resource_id > 0), only count actual group chat members
+                ResourceMember.copied_resource_id == 0,
+            )
+            .group_by(ResourceMember.resource_id)
+            .all()
+        )
+        member_counts = {row[0]: row[1] for row in member_count_results}
```

Ruff (0.15.5)

[error] Local variable `member_counts` is assigned to but never used

Remove assignment to unused variable `member_counts`

(F841)